app/models/dialog_field_text_box.rb

```diff
@@ -25,6 +25,7 @@ def value_from_dialog_fields(dialog_values)
   end
 
   def automate_output_value
+    return nil if @value.nil?
```

Comment:
@eclarizio Will returning a `nil` cause an issues for the UI or Self-service UI that you are aware of?